src/store/ConfigStore.ts

```diff
@@ -19,7 +19,7 @@ export class ConfigStore extends VuexModule {
           }
 
           return {
-            ...radar, selectLabel: displayLabel
+            ...radar, displayLabel: displayLabel
```

Comment:
@niconoe I hope this same variable name `displayLabel` doesn't cause any issues. It works on my local test.

Reply:
I changed the name to `displayLabel`, since it is not only used in the select dropdown